packages/visual-dev/src/components/RadioCard/RadioCard.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import root from 'react-shadow/styled-components'
+import styled from 'styled-components'
+import * as Styles from './Styles'
+import { IconKey, Icon } from '../Icon'
+
+type InputProps = OptionProps & React.ComponentProps<'input'>
+
+interface OptionProps {
+  value?: any
+  onChange?: any
+  options?: any
+  icon?: IconKey
+}
+
+const ShadowDom = styled(root.div)``
+
+const RadioLabel = styled.label<{ isChecked: boolean }>`
+  ${Styles.RadioLabelStyle}
+  ${(props) => (props.isChecked ? 'border: 2px solid #DC8F32;' : '&:hover {border: 2px solid #858585;}')}
+`
+const RadioInput = styled.input`
+  ${Styles.RadioInputStyle}
+`
+
+const RightSegment = styled.div`
+  ${Styles.RightSegmentStyle}
+`
+
+const RadioText = styled.div`
+  ${Styles.RadioTextStyle}
+`
+
+const LeftSegment = styled.div<{ isChecked: boolean }>`
+  ${Styles.LeftSegmentStyle}
+  ${(props) => (props.isChecked ? 'color: #DC8F32;' : '')}
+`
+
+export const RadioCard = React.forwardRef<React.ElementRef<'input'>, InputProps>((props, forwardedRef) => {
+  const { value, onChange, options, icon = 'calendar', ...rest } = props
+
+  let icon_color
+  value ? (icon_color = '#DC8F32') : ''

```

Comment:
Ideally we shouldn't have any hard coded colours (prefer CSS variables). `#DC8F32` especially is fairly common across the portal I believe. Ideally the other hard coded colours in this file should also be made to be variables.

Reply:
```
 value ? (icon_color = 'var(--sq-action-primary-hovered)') : ''
```
done!